src/core/utils.js

```diff
@@ -40,8 +40,23 @@ export function convertBoolean(value) {
   }
 }
 
-export function getClientApp() {
-  // TODO: Look at user-agent header passed in.
+/*
+ * This is a very simplistic check of the user-agent string in order to redirect to
+ * the right set of AMO data.
+ *
+ * More complete UA detection for compatibility will take place elsewhere.
+ *
+ */
+export function getClientApp(userAgentString) {
+  if (userAgentString && userAgentString.includes) {
+    // We are going to return android as the application if it's *any* android browser.
+    // whereas the previous behaviour was to only return 'android' for FF Android.
+    // This way we are showing more relevant content, and if we prompt for the user to download
+    // firefox we can prompt them to download Firefox for Android.
+    if (userAgentString.includes('Android')) {
```

Comment:
All the strings I found seem to have 'Android' title-case but it won't hurt to make it check lower-case .